server/tests/services/foods/foods.test.ts

```diff
@@ -0,0 +1,109 @@
+import mongoose from "mongoose";
+import { singleTestFood, foodKeys, manyTestFoods } from "./constants";
+import { FoodModel, FoodDocument } from "../../../src/models/Food";
+import FoodService from "../../../src/services/food";
+import { expect, test } from "@jest/globals";
+import { pick } from "lodash";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const CONNECT_SUCCESS = "Connected to MongoDB";
+async function connectToDB(): Promise<string> {
+  return mongoose
+    .connect(process.env.CONNECTION_URL!)
+    .then(() => CONNECT_SUCCESS);
+}
+
+test("Connects to MongoDB", async () => {
+  const connectionResult = await connectToDB();
+  expect(connectionResult).toBe(CONNECT_SUCCESS);
+});
+
+test("Creates a food", async () => {
+  const createdFood = await FoodService.create(singleTestFood);
+  // ensure not null
+  expect(createdFood).toBeDefined();
+  // ensure input match
+  expect(pick(createdFood, foodKeys)).toStrictEqual(singleTestFood);
+  // ensure date not null
+  expect(createdFood.created_at).toBeDefined();
+
+  // delete test document
+  if (!!createdFood) {
+    await FoodModel.deleteOne({ _id: createdFood._id });
+  }
+});
+
+test("Creates many foods", async () => {
+  const createdFoods: FoodDocument[] = await FoodService.createMany(
+    manyTestFoods
+  );
+  // ensure creation
+  expect(createdFoods.length).toBe(manyTestFoods.length);
+
+  createdFoods.forEach((createdFood, index) => {
+    // ensure equality
+    expect(pick(createdFood, foodKeys)).toStrictEqual(manyTestFoods[index]);
+    // ensure date
+    expect(createdFood.created_at).toBeDefined();
+  });
+
+  // delete test documents
+  const createdFoodsId = createdFoods.map((food) => food._id);
+  await FoodModel.deleteMany({
+    _id: { $in: createdFoodsId },
+  });
+});
+
+test("Finds a food by ID", async () => {
+  const createdFood: FoodDocument = await FoodModel.create({
+    ...singleTestFood,
+    created_at: new Date(),
+  });
+  const createdFoodId = createdFood._id;
+
+  const foundFood: FoodDocument | null = await FoodService.findFoodById(
+    createdFoodId
+  );
+
+  // ensure not null
+  expect(foundFood).toBeDefined();
+  // ensure input match
+  expect(JSON.stringify(foundFood)).toBe(JSON.stringify(createdFood));
```

Comment:
May be useful to mention why we need to JSON.stringify here.

Reply:
on it